### PR TITLE
Show invest price impact when it is less than 0.01% (Closes #1762)

### LINKED ISSUE #1762: Invest flow: update price impact format when it is less than 0.01%
**Describe the bug**
Currently, in the invest flow, when the price impact is less than 0.01%, the price impact is listed at 0.00%. This may make some users feel like it is not working properly. 

Screen Shot 2022-04-21 at 9 51 12 PM

**To Reproduce**
1. Go to the Invest flow
2. Go to add liquidity on a pair like reth/weth

**Expected behavior**
If the price impact is greater than 0 and less than 0.01%, the price impact displayed should be `< 0.01%`. This will help users to understand that the calculation is indeed working but that the impact is tiny.

### PR DESCRIPTION
**Pull request by alter-eggo**
# Description
This pr closes https://github.com/balancer-labs/frontend-v2/issues/1762 bug and improves invest price impact. It is shown as '< 0.01%' if less than 0.01% and greater than 0.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [x] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have requested at least 2 reviews (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [ ] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/composables/useNumbers.ts [hunk 1]
```diff
@@ -14,13 +14,15 @@ export interface FNumOptions extends Intl.NumberFormatOptions {
   fixedFormat?: boolean; // If true, don't auto-adjust based on number magnitde
   abbreviate?: boolean; // If true, reduce number size and add k/M/B to end
   dontAdjustLarge?: boolean; // If true, don't auto-adjust if the number is large
+  returnLessBasicPoint?: boolean; // If true, return less than basic point (< 0.01%) instead of 0.00%, if num < 0.00005
```
Comment (evgenyboxer) [inline src/composables/useNumbers.ts:17]: I think you meant `basis` point yeah?

File: src/composables/useNumbers.ts [hunk 2]
```diff
@@ -164,6 +166,10 @@ export default function useNumbers() {
       number = 0;
     }
 
+    if (options.returnLessBasicPoint && number > 0 && number < 0.00005) {
+      return '< 0.01%';
```
Comment (evgenyboxer) [inline src/composables/useNumbers.ts:170]: I think we should prob be consistent and always return it without passing a config option.
Thoughts @garethfuller?

File: src/composables/useNumbers.ts (hunk 1 above)
Comment (alter-eggo) [reply src/composables/useNumbers.ts:17]: yea, thank you. I meant `basis` point

**Review by garethfuller — CHANGES_REQUESTED** [review]

**Comment by garethfuller** [comment]
I needed to move the conditional within the isPercentage block so it didn't affect all number formatting. Closing in favour of #1834

**Pull request closed without merging**